proto/api/v1/tenant.proto

```diff
@@ -398,3 +404,20 @@ message TenantServiceUpdateMemberResponse {
   // TenantMember is the updated membership
   TenantMember tenant_member = 1;
 }
+
+// TenantServiceRequestAdmissionRequest is used to request admission for a tenant
+message TenantServiceRequestAdmissionRequest {
+  // Login of the tenant for whom admission is requested
+  string login = 1;
+  // Name of the user
+  string name = 2;
+  // Email of the user
+  string email = 3 [(buf.validate.field).string.email = true];
+  // AcceptedTermsAndConditions indicates if the user has accepted the terms and conditions
+  bool accepted_terms_and_conditions = 4;
+  // EmailConsent indicates if the user gave consent to receive emails
+  bool email_consent = 5;
```

Comment:
Email consent is only used in the hubspotservice, which needs a paymentCustomer.
We can create a empty payment customer, which takes in only email and name, this won't break anything right?
`err = t.hubspotService.CreateNewHubspotContact(ctx, customer, req.EmailConsent)`

Reply:
I wonder why we are passing an entire payment customer to the `CreateHubspotContact` method, when the only two values it needs are `name` and `email`. Maybe instead of creating a dummy customer we should refactor the `CreateHubspotContact` method to accept `name string` and `email string` instead of a payment customer.